Test/CMOR_input_TestTables.json

```diff
@@ -37,7 +37,7 @@
     "grid_label":             "gn",
     "nominal_resolution":     "10000 km",
  
-    "license":                 "CMIP6 model data produced by Lawrence Livermore PCMDI is licensed under a Creative Commons Attribution ShareAlike 4.0 International License (https://creativecommons.org/licenses). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file) and at https:///pcmdi.llnl.gov/. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
+    "license":                 "CMIP6 model data produced by Lawrence Livermore PCMDI is licensed under a Creative Commons Attribution-ShareAlike 4.0 International License (https://creativecommons.org/licenses/by-sa/4.0/). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file) and at https:///pcmdi.llnl.gov/. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
```

Comment:
@mauzey1 I wonder whether we should update this to the `CC BY 4.0` (not the `-SA` variant) to be consistent, even though this is a test
https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_license.json#L3

Reply:
The compressed version of this license string is "Creative Commons Attribution 4.0 International (CC BY 4.0; https://creativecommons.org/licenses/by/4.0/) License"

https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_source_id.json#L19

Reply:
Do you want this change applied to all the CMOR user input used by tests?  Or just this file?

Reply:
@durack1 Shouldn't the string be "Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/)"?

Reply:
@mauzey1 the license identifier is "Creative Commons Attribution 4.0 International" and the info in parentheses points to details, so I am happy with the formatting.

Regarding the extra work, if this won't impact any of the tests, then I suggest we only change a small number - if you foresee it causing problems if it is inconsistent, then the extra work is warranted